PilotDataPlatform/metadata
# PILOT-1482 Implement POST endpoint for creating favourites

## Summary

- Add `POST /v1/favourite` endpoint for creating favourite items/collections, as well as all models and migrations required by it.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1482

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Manually create some favourites and check the database to see how they're written.
2. Review the [API design](https://indocconsortium.atlassian.net/wiki/spaces/PILOT/pages/3189309472/Favourites+API+design) and ensure that the implementation matches the design.
